desktop-widgets/filterwidget2.cpp

```diff
@@ -0,0 +1,8 @@
+#include "filterwidget2.h"
+
+FilterWidget2::FilterWidget2(QWidget* parent)
+: QWidget(parent)
+, ui(new Ui::FilterWidget2())
+{
+    ui->setupUi(this);
```

Comment:
oh good, first line of code that isn't tool generated, and indentation is done with spaces instead of tabs...